cfgov/unprocessed/js/routes/on-demand/feedback-form.js

```diff
@@ -3,8 +3,8 @@
    ========================================================================== */
 
 
-const COMMENT_ERRORS = require( '../../config/error-messages-config' ).COMMENT || {};
-const OPTION_ERRORS = require( '../../config/error-messages-config' ).OPTION || {};
+const COMMENT_ERRORS = require( '../../config/error-messages-config' ).COMMENT;
```

Comment:
Was there a scenario where this fallback was needed? Should this check the value of `COMMENT_ERRORS` further down and set it to an empty object if it is undefined?

Reply:
It's a config file; why would it not return an object?

Reply:
Yeah it doesn't look needed to me, but it was also added. Just not sure if there was a reason.